stockflux-securities-master/src/enums.js

```diff
@@ -1,6 +1,7 @@
 export const TableState = {
   LOADING: "loading",
   DELETING: "deleting",
+  UPDATING: "updating",
```

Comment:
How's updating different from loading? If they are doing the same thing, maybe we should just keep one, remove redundant one and rename it so it would accommodate both usage scenarios?

Reply:
Updating and Deleting do similar thing but display different messages.

Reply:
Loading produces a large spinner in place of the securities, deleting and updating produce a spinner underneath the securities table in place of the alerts. It's a bit hard to find a name that incorporates both updating and deleting though because they are pretty different concepts. Maybe modifying?

Reply:
Yeah, or maybe just keep it as `UPDATING`, as deleting is also updating in a way?